### PR TITLE
feat: duplicate field with cell data

### PR DESCRIPTION
**Pull request by Xazin**
Closes: #4092 

### Feature Preview

https://github.com/AppFlowy-IO/AppFlowy/assets/42929161/cad36f65-c314-485f-9728-539ca38683bc

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/resources/translations/en.json [hunk 1]
```diff
@@ -563,6 +563,7 @@
       "insertLeft": "Insert Left",
       "insertRight": "Insert Right",
       "duplicate": "Duplicate",
+      "duplicateWithData": "Duplicate w/ data",
```
Comment (LucasXu0) [inline frontend/resources/translations/en.json:566]: The abbreviation seems a bit confusing.

File: frontend/rust-lib/flowy-database2/src/services/database/database_editor.rs [hunk 2]
```diff
@@ -441,8 +446,23 @@ impl DatabaseEditor {
...
+          let new_field_id = duplicated_field.id.clone();
+          let new_row_id = cell.row_id;
+
+          if let Some(new_cell) = new_cell {
+            self
+              .update_cell(view_id, new_row_id, &new_field_id, new_cell)
+              .await?;
+          }
+        }
+      }
```
Comment (LucasXu0) [inline frontend/rust-lib/flowy-database2/src/services/database/database_editor.rs:465]:
```suggestion
      if duplicate_data {
        let new_field_id = duplicated_field.id.clone();
        let cells = self.get_cells_for_field(view_id, field_id).await;
        for cell in cells {
          if let Some(new_cell) = cell.cell.clone() {
            self
              .update_cell(view_id, cell.row_id, &new_field_id, new_cell)
              .await?;
          }
        }
      }
```

**Comment by richardshiue** [comment]
Let's just make this the default behavior. I'm not very high on too many options with little differences haha

**Comment by LucasXu0** [comment]
similar issue to the mobile CI job, no disk space.

https://github.com/AppFlowy-IO/AppFlowy/actions/runs/8062711655

**Pull request merged**